vars/default.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Put internal variables here with values for unrecognized distributions.
+
+# Example:
```

Comment:
I think you should delete the `# Example` line. It is not an example anymore, but an actual working code.